src/styling/projects-page-styling.css

```diff
@@ -31,12 +96,52 @@
 }
 
 .scrolling {
-  height: 85vh;
+  height: 90vh;
```

Comment:
just want to double check will this scroll the same on all screen sizes?

Reply:
The scroll is only relevant for the large (or xl) view because on smaller screens you don't get the side by side view since it would be too squished; instead you have the the description card which you x out of to see other cards.